docs/governance/contribute/git-guideline.md

```diff
@@ -0,0 +1,109 @@
+# Git 提交指南
+
+本文介绍了Git提交变更时需要注意的事项，如果拒绝接受本文的内容会导致提交的变更无法被接受。
+
+## 1. 关于 issue
+
+在提交一个issue之前，请先查阅已经关闭的issue，也许在关闭的issue中已经存在合适的解决方案。
+
+如果没有找到合适的方案，我们提供了4种模版在创建issue的时候使用。
+- Bug Report : 发现了一个Bug，可以通过 Bug Report 模版创建 issue 与我们联系。
+- Enhancement : 开发者对工具进行了增强，可以通过 Enhancement 模版创建 issue 来介绍增加的内容。
+- Feature Request : 在使用的过程中想要为工具增加某些新的特性或者功能，可以通过 Feature Request 模版创建 issue 来描述新特性。
+- Ask a Question : 如果有任何的疑问，可以通过 Ask a Question 模版来创建一个 issue 与我们联系。
+
+在选择合适的模版后，只需要填写模版上的要求填写的内容即可。如果在创建issue的时候发现没有模版，或者模版内容为空，可以通过微信群，钉钉群或者邮件向我们反馈这个问题。
+
+## 2. 关于Git分支
+我们参考[Angular规范](https://github.com/angular/angular/blob/main/CONTRIBUTING.md)为分支命名。
+推荐的格式如下：
+```
+{type}/{your_account_ID}/{a_short_description}
+```
+分支名称主要包括三个字段，并通过 “/” 分割。其中：
+ - {type} : 当前分支内容的类型。
+ - {your_account_ID}: 进行这次修改的账户ID。
+ - {a_short_description}: 一个简短的描述，介绍这个分支的主要内容。
+
+ e.g. 张三创建了一个修复了输出格式化bug的分支，并打算使用账户 zhangS 提交 mr，对应的分支名称`fix/zhangS/fix_output_fmt_bug`。
+
+## 3. 关于Git Commit
+我们参考[Commitizen](https://github.com/commitizen/cz-cli)书写Commit Message。
+```
+注: 如果直接使用Commitizen生成Commit Message，需要注意因为Commitizen是开发人员
+管理commit的工具，与项目本身无关联，因此由Commitizen生成的中间产物(如: node_modules文件
+目录)可能没有在项目.gitignore文件中。
```

Comment:
You can give the solution, e.g. , "If you want to create a commit message by commitizen, add the following code snippet to .gitignore".
```
# commitizen
package.json
package-lock.json
node_modules/*
```